src/main/java/com/synopsys/integration/alert/channel/email/EmailChannel.java

```diff
@@ -146,9 +156,13 @@ public void sendMessage(EmailProperties emailProperties, Set<String> emailAddres
         return optionalAttachmentFile;
     }
 
-    private String createEnhancedSubjectLine(String originalSubjectLine, String providerProjectName) {
+    private String createEnhancedSubjectLine(String originalSubjectLine, String providerProjectName, Optional<String> providerProjectVersionName) {
         if (StringUtils.isNotBlank(providerProjectName)) {
-            return String.format("%s | For: %s", originalSubjectLine, providerProjectName);
+            String subjectLine = String.format("%s | For: %s", originalSubjectLine, providerProjectName);
+            if (providerProjectVersionName.isPresent()) {
+                subjectLine += String.format(" %s", providerProjectVersionName.get());
+            }
+            return subjectLine;
```

Comment:
We might want to consider truncating this if it exceeds a certain number of characters.
78 is the suggested number of RCF 2822 https://tools.ietf.org/html/rfc2822#section-2.1.1

Reply:
String utils should have the abbreviate method if I remember correctly to do that.